common/factories/orders.js

```diff
@@ -1,7 +1,215 @@
-Factory.define('orderForAF', ReactionCore.Collections.Orders,
+Factory.define('orderForAF', Orders,
   Factory.extend('order', {
-    shopId: ReactionCore.getShopId(),
     shipping: [{address: faker.reaction.address()}],
     billing: [{address: faker.reaction.address()}]
   })
 );
+
```

Comment:
I'd like to randomize this (and all) factory a little more so that it's testing various colors, sizes, etc as well. We may have to add some faker functionality to accomplish this.